diff -Naur app/src/downloader.cpp app2/src/downloader.cpp
--- src/downloader.cpp	2020-04-17 13:19:20.000000000 +0000
+++ src/downloader.cpp	2020-04-18 02:08:39.000000000 +0000
@@ -247,7 +247,15 @@
     std::string email;
     std::string password;
 
-    if (!Globals::globalConfig.sEmail.empty() && !Globals::globalConfig.sPassword.empty())
+    char const* envEmail = getenv( "GOG_USER" );
+    char const* envPass = getenv( "GOG_PASSWORD" );
+
+    if ( envEmail != NULL && envPass != NULL ) 
+    {
+	email = strdup(envEmail);
+	password = strdup(envPass);
+    } 
+    else if (!Globals::globalConfig.sEmail.empty() && !Globals::globalConfig.sPassword.empty())
     {
         email = Globals::globalConfig.sEmail;
         password = Globals::globalConfig.sPassword;
